code/modules/projectiles/ammunition/energy/laser.dm

```diff
@@ -9,7 +9,7 @@
 
 /obj/item/ammo_casing/energy/lasergun
 	projectile_type = /obj/projectile/beam/laser
-	e_cost = 83
+	e_cost = 71
```

Comment:
More than just the laser use this ammo casing. This is also a pretty big buff to the captain's gun and the laser arm implant.

Reply:
Hm! not wrong. I guess i'll have to make a subtype that is cheaper, eh?